<!-- MHonArc v2.6.18 -->
<!--X-Subject: [sup&#45;talk] exception on start after ruby upgrade -->
<!--X-From-R13: cnagf <cnagfNpf.uzp.rqh> -->
<!--X-Date: Tue, 8 Nov 2011 11:31:40 &#45;0800 (PST) -->
<!--X-Message-Id: 1320775126&#45;sup&#45;274@WOPR -->
<!--X-Content-Type: text/plain -->
<!--X-Head-End-->
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
        "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>[sup-talk] exception on start after ruby upgrade</title>
<link rev="made" href="mailto:pants@cs.hmc.edu">
</head>
<body>
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<hr>
[<a href="msg00125.html">Date Prev</a>][<a href="msg00040.html">Date Next</a>][<a href="msg00125.html">Thread Prev</a>][<a href="msg00009.html">Thread Next</a>][<a href="maillist.html#00297">Date Index</a>][<a href="threads.html#00297">Thread Index</a>]
<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<h1>[sup-talk] exception on start after ruby upgrade</h1>
<hr>
<!--X-Subject-Header-End-->
<!--X-Head-of-Message-->
<ul>
<li><em>To</em>: sup-talk &lt;<a href="mailto:sup-talk%40rubyforge.org">sup-talk@rubyforge.org</a>&gt;</li>
<li><em>Subject</em>: [sup-talk] exception on start after ruby upgrade</li>
<li><em>From</em>: pants &lt;<a href="mailto:pants%40cs.hmc.edu">pants@cs.hmc.edu</a>&gt;</li>
<li><em>Date</em>: Tue, 08 Nov 2011 10:01:02 -0800</li>
<li><em>Authentication-results</em>: mx.google.com; spf=pass (google.com: domain of <a href="mailto:sup-talk-bounces%40rubyforge.org">sup-talk-bounces@rubyforge.org</a> designates 205.234.109.19 as permitted sender) smtp.mail=<a href="mailto:sup-talk-bounces%40rubyforge.org">sup-talk-bounces@rubyforge.org</a></li>
<li><em>Delivered-to</em>: <a href="mailto:eg%40gaute.vetsj.com">eg@gaute.vetsj.com</a></li>
<li><em>Delivered-to</em>: <a href="mailto:sup-talk%40rubyforge.org">sup-talk@rubyforge.org</a></li>
<li><em>List-archive</em>: &lt;<a href="http://rubyforge.org/pipermail/sup-talk">http://rubyforge.org/pipermail/sup-talk</a>&gt;</li>
<li><em>List-help</em>: &lt;<a href="mailto:sup-talk-request@rubyforge.org?subject=help">mailto:sup-talk-request@rubyforge.org?subject=help</a>&gt;</li>
<li><em>List-id</em>: User &amp; developer discussion of Sup &lt;sup-talk.rubyforge.org&gt;</li>
<li><em>List-post</em>: &lt;<a href="mailto:sup-talk@rubyforge.org">mailto:sup-talk@rubyforge.org</a>&gt;</li>
<li><em>List-subscribe</em>: &lt;<a href="http://rubyforge.org/mailman/listinfo/sup-talk">http://rubyforge.org/mailman/listinfo/sup-talk</a>&gt;,	&lt;<a href="mailto:sup-talk-request@rubyforge.org?subject=subscribe">mailto:sup-talk-request@rubyforge.org?subject=subscribe</a>&gt;</li>
<li><em>List-unsubscribe</em>: &lt;<a href="http://rubyforge.org/mailman/options/sup-talk">http://rubyforge.org/mailman/options/sup-talk</a>&gt;,	&lt;<a href="mailto:sup-talk-request@rubyforge.org?subject=unsubscribe">mailto:sup-talk-request@rubyforge.org?subject=unsubscribe</a>&gt;</li>
<li><em>Sender</em>: <a href="mailto:sup-talk-bounces%40rubyforge.org">sup-talk-bounces@rubyforge.org</a></li>
<li><em>User-agent</em>: Sup/0.12.1</li>
</ul>
<!--X-Head-of-Message-End-->
<!--X-Head-Body-Sep-Begin-->
<hr>
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<pre>this morning I upgraded to ruby 1.9.3 on my system running archlinux.  I
reinstalled/rebuilt all of my ruby gems.  sup proceeded to throw an
exception on attempting to start it.  the log follows:

--- RuntimeError from thread: main
wrong id called on {&quot;uri&quot;=&gt;&quot;maildir:/home/pants/mail/inbox/&quot;, &quot;usual&quot;=&gt;true, &quot;archived&quot;=&gt;false, &quot;id&quot;=&gt;1, &quot;labels&quot;=&gt;[]}
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup.rb:18:in `id'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup/source.rb:207:in `block (2 levels) in load_sources'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup/source.rb:207:in `map'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup/source.rb:207:in `block in load_sources'
/usr/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup/source.rb:206:in `load_sources'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup/util.rb:609:in `method_missing'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup/index.rb:92:in `load'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/lib/sup/util.rb:609:in `method_missing'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/bin/sup:154:in `&lt;module:Redwood&gt;'
/usr/lib/ruby/gems/1.9.1/gems/sup-0.12.1/bin/sup:69:in `&lt;top (required)&gt;'
/usr/bin/sup:19:in `load'
/usr/bin/sup:19:in `&lt;main&gt;'

thanks.
-- 
jaron `pants' kent-dobias
pants@cs.hmc.edu
425 999 1948

_______________________________________________
sup-talk mailing list
sup-talk@rubyforge.org
<a  rel="nofollow" href="http://rubyforge.org/mailman/listinfo/sup-talk">http://rubyforge.org/mailman/listinfo/sup-talk</a>
</pre>
<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
<hr>
<!--X-Follow-Ups-End-->
<!--X-References-->
<!--X-References-End-->
<!--X-BotPNI-->
<ul>
<li>Prev by Date:
<strong><a href="msg00125.html">Re: [sup-talk] exception --- RuntimeError from thread: poll after	loading inbox</a></strong>
</li>
<li>Next by Date:
<strong><a href="msg00040.html">Re: [sup-talk] exception --- RuntimeError from thread: poll after loading inbox</a></strong>
</li>
<li>Previous by thread:
<strong><a href="msg00125.html">Re: [sup-talk] exception --- RuntimeError from thread: poll after	loading inbox</a></strong>
</li>
<li>Next by thread:
<strong><a href="msg00009.html">[sup-talk] failure to start sup</a></strong>
</li>
<li>Index(es):
<ul>
<li><a href="maillist.html#00297"><strong>Date</strong></a></li>
<li><a href="threads.html#00297"><strong>Thread</strong></a></li>
</ul>
</li>
</ul>

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
</body>
</html>
